### PR TITLE
Release versions for 3.2.0

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

**Comment by oleorhagen** [comment]
https://github.com/mendersoftware/meta-mender/pull/1573

Find mender-configure above

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

File: meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.0.bb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require mender-connect.inc
+
+RDEPENDS_${PN} = "glib-2.0 mender-client (>= 2.5)"
```
Comment (lluiscampos) [inline meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.0.bb:3]: This is not correct, `mender-connect` now requires `mender-client (>= 3.2)` which is already the default in the `.inc` file. So this line must be removed.
```suggestion
```

File: meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.0.bb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+SRC_URI = "git://github.com/mendersoftware/mender-connect.git;protocol=https;branch=2.0.x"
+
+# Tag: 2.0.0
+SRCREV = "377062297285d84908285bcddab8aaeaa8e4d41c"
+
+# Enable this in Betas, and in branches that cannot carry this major version as
+# default.
+# Downprioritize this recipe in version selections.
+#DEFAULT_PREFERENCE = "-1"
```
Comment (lluiscampos) [inline meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.0.bb:21]: This is a major version increase so we need to down prioritize it in `dunfell` (obs! not in `master`) same way we are doing for `mender-client` > 3.0

File: meta-mender-core/recipes-mender/mender-client/mender-client_3.2.0.bb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+SRC_URI = "git://github.com/mendersoftware/mender;protocol=https;branch=3.2.x"
+
+# Tag: 3.2.0
+SRCREV = "f6e9401329ff26e27955651a20f9025275c717bf"
+
+# Enable this in Betas, and in branches that cannot carry this major version as
+# default.
+# Downprioritize this recipe in version selections.
+# DEFAULT_PREFERENCE = "-1"
```
Comment (lluiscampos) [inline meta-mender-core/recipes-mender/mender-client/mender-client_3.2.0.bb:19]: Needs to be down-priotized like the rest of the `mender-client` > 3.0 recipes in `dunfell`

**Review by lluiscampos — CHANGES_REQUESTED** [review]

**Review by lluiscampos — COMMENTED** [review]
Extreme nit-pick: two commits have "add ..." in the changelog entry (title), the rest have "Add ..." :smiley:

File: meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.0.bb [hunk 4]
```diff
@@ -0,0 +1,30 @@
+require mender-connect.inc
+
+RDEPENDS_${PN} = "glib-2.0 mender-client (>= 3.2)"
```
Comment (lluiscampos) [inline meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.0.bb:3]: I think it is better to remove this line (as suggested before, maybe wasn't clear?). `RDEPENDS_` is already defined in `mender-connect.inc` and does not need to be overridden (unlike the 1.x recipes that need to override the version of mender-client to depend on).

**Review by lluiscampos — APPROVED** [review]

File: meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.0.bb (hunk 4 above)
Comment (oleorhagen) [reply meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.0.bb]: Fixed

**Pull request merged**